css/contact.css

```diff
@@ -0,0 +1,142 @@
+===================================
+===================================
+*/
+/* 
+===================================
+Page Component Styles (icon grid, team member card)
+===================================
+*/
+
+/* contact form */
+.wrapper {
+	width: var(--mobile-wrapper-width);
+}
+.form-input {
+	display: flex;
+	flex-direction: column;
+	justify-content: space-around;
+	/* padding: 20px; */
+}
+input[type="text"],
+[type="email"],
+select,
+textarea {
+	resize: vertical;
+	font-family: var(--main-font);
+	font-size: var(--font16);
+	font-weight: var(--font-weight-regular);
+	color: var(--color-onyx);
+	background-color: var(--color-white);
+	border: 1px solid #6c6c6c;
+	width: 100%;
+	border-radius: 5px;
+	padding: 10px;
+	margin: 5px 0;
+}
+.message {
+	min-height: 140px;
+	width: 100%;
+	border-radius: 10px;
+	border: none;
+	margin: 10px 0;
+	align-self: flex-start;
+}
+.button2 {
+	border-radius: 10px;
+	background-color: var(--button2-color);
+	padding: 10px 40px;
+	color: var(--color-white);
+	border: none;
+	min-width: 187px;
+	min-height: 50px;
+	margin: 20px;
+	font-weight: var(--font-weight-bold);
+	font-size: var(--font21);
+}
+.button2:hover {
+	background-color: var(--color-white);
+	color: var(--color-onyx);
+}
+/* 
+===================================
+Section Styles (background, padding, etc.)
+===================================
+*/
+body {
+	background: var(--color-bluepurple);
+	color: var(--color-white);
+}
+h3 {
+	color: var(--color-white);
+	text-align: center;
+}
+p {
+	color: var(--color-white);
+	gap: 20px;
+}
+
+/* 
+===================================
+Media Queries (overriding styles for different screen sizes)
+
+Note: Here we are using a mobile-first approach, so we start with the smallest screen size and work our way up.
+Start by writing styles for the mobile version of the design on the lines above.
+Anything that needs to change for larger screens should be added to the media queries below.
+
+Feel free to change the min/max-width values to whatever you need. Use variables.
+===================================
+*/
+
+/* Tablet and Larger */
+@media screen and (min-width: 768px) {
+	/* Add styles that apply to table and desktop */
+	.contact-form-parent {
+		display: flex;
+		flex-direction: row;
+		justify-content: space-between;
+		align-items: center;
+		text-align: left;
+		margin: 100px 150px;
+		gap: 100px;
+	}
+	.form-info {
+		width: 35%;
+	}
+	.form-info h3 {
+		text-align: left;
+	}
+	.wrapper {
+		width: var(--desktop-wrapper-width);
+	}
+}
+
+/* Tablet Only */
+@media screen and (min-width: 768px) and (max-width: 1024px) {
+	/* Add styles that only apply to tablet. These are usually fixes to get resposnivenes right */
+	.contact-form-parent {
+		margin: 100px 75px;
+	}
+	.form-info {
+		width: 50%;
+	}
+}
+
+/* Desktop Only */
+@media screen and (min-width: 1024px) {
+	/* Add styles that apply specifically to desktop */
+	.form-info h3 {
```

Comment:
The vertical centering between the image and form looks slightly off. Remove the top margin from the h3.

<img width="1656" alt="Screen Shot 2024-01-11 at 1 04 27 PM" src="https://github.com/indigenousfriends/itlct-website-redesign/assets/704697/8460a318-63de-493d-92cd-993dea2deb50">

Reply:
This is close, but the text is getting scrunched up on desktop now:
<img width="865" alt="Screen Shot 2024-01-18 at 2 09 47 PM" src="https://github.com/indigenousfriends/itlct-website-redesign/assets/704697/2484e6d8-0b06-4c28-8f64-ef31291e5a78">

Reply:
I see that you made some updates to this, however, it's still getting  scrunched on desktop. It looks good when I remove the padding from both `.form-info` and `.contact-form-parent` in the inspector. 

<img width="1348" alt="Screen Shot 2024-01-19 at 1 37 11 PM" src="https://github.com/indigenousfriends/itlct-website-redesign/assets/704697/f20bd2ca-71a8-45f3-813c-c4d6f62e9022">
<img width="730" alt="Screen Shot 2024-01-19 at 1 35 32 PM" src="https://github.com/indigenousfriends/itlct-website-redesign/assets/704697/8b83bce5-63b9-4271-a0bb-ee20b3010586">


Please give it another look and let me know if you are getting something different.